WordPress/Classes/Extensions/UIButton+Extensions.swift

```diff
@@ -0,0 +1,27 @@
+import UIKit
+
+extension UIButton {
+    /// Creates a bar button item that looks like the native title menu
+    /// (see `navigationItem.titleMenuProvider`, iOS 16+).
+    static func makeMenuItem(title: String, menu: UIMenu) -> UIButton {
```

Comment:
Wdyt of renaming this to makeFilterButton or makeDropDownButton? MenuItem makes me think of an individual item in a menu

Reply:
`UIButton.makeMenu` sounds good!